client/src/components/Home.tsx

```diff
@@ -173,67 +174,69 @@ function Home() {
+                <p
+                  className={css`
                   font-size: 0.875rem;
                   line-height: 1.5;
                 `}
-              >
-                Mirlo is under construction. If you'd like to contribute check
-                out{" "}
-                <a href="https://github.com/funmusicplace/mirlo/">
-                  the code on GitHub
-                </a>
-                , <a href="https://discord.gg/VjKq26raKX">join our Discord</a>,
-                or <a href="mailto:mirlodotspace@proton.me">email us</a>.
-              </p>
+                >
+                  <Trans
+                    t={t}
+                    i18nKey="mirloConstruction"
+                    components={{
+                      github: <a href="https://github.com/funmusicplace/mirlo/"></a>,
+                      discord: <a href="https://discord.gg/VjKq26raKX"></a>,
+                      email: <a href="mailto:mirlodotspace@proton.me"></a>,
+                    }}
+                  />
+                </p>
+              </div>
             </div>
           </div>
-        </div>
-      )}
-      {user && posts.length > 0 && (
-        <>
-          <h2
-            className={css`
+        )}
+        {user && posts.length > 0 && (
+          <>
+            <h2
+              className={css`
               margin-top: 1rem;
               margin-bottom: 1rem;
             `}
-          >
-            Latest posts from the community:
-          </h2>
-          {posts.map((p) => (
-            <Box
-              key={p.id}
-              className={css`
+            >
+              {t("latestCommunityPost")}
+            </h2>
+            {posts.map((p) => (
+              <Box
+                key={p.id}
+                className={css`
                 margin-bottom: 1rem;
 
                 h3 {
                   padding-bottom: 0.4rem;
                 }
               `}
-            >
-              <h3>
-                <Link to={`/post/${p.id}/`}>{p.title}</Link>
-              </h3>
-              {p.artist && (
-                <em>
-                  by{" "}
-                  <Link to={`/${p.artist.urlSlug ?? p.artist.id}`}>
-                    {p.artist?.name}
-                  </Link>
-                </em>
-              )}
-              <PostContent content={p.content} />
-            </Box>
-          ))}
-        </>
-      )}
-    </div>
-  );
-}
+              >
+                <h3>
+                  <Link to={`/post/${p.id}/`}>{p.title}</Link>
+                </h3>
+                {p.artist && (
+                  <em>
+                    by{" "}
+                    <Link to={`/${p.artist.urlSlug ?? p.artist.id}`}>
+                      {p.artist?.name}
+                    </Link>
+                  </em>
+                )}
+                <PostContent content={p.content} />
+              </Box>
+            ))}
+          </>
+        )}
+      </div>
+    );
+  }
```

Comment:
Is there a reason for these indentations?

Might be useful to have prettier installed in your code editor to convert on save, that way it won't create these large diffs. Though it would be even better if it was installed in the repo. I'll fix that.